examples/4_Sustainability/4-1_BoM_Sustainability_summary.py

```diff
@@ -135,10 +151,32 @@ def plot_impact(df, title, textinfo="percent+label", hoverinfo="value+name"):
         for item in sustainability_summary.transport_details
     ]
 )
+transport_df_full.describe()
+# -
+
+# Most of these transport stages contribute little to the overall sustainability impact. To make a visualization more
+# insightful, group all transport stages that contribute less than 5% of embodied energy or climate change in a single
+# 'Other' transport stage.
+
+# +
+# Define the criterion
```

Comment:
We could consider adding this to the library. We could include a 'get_aggregated_transport_details()` method or something which accepts a threshold and does the filtering.